docker/docker-compose.mailpit.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  mailpit:
+    container_name: mailhog
```

Comment:
why the container name is mailhog?

Reply:
For backwards compatibility so you can still reference "mailhog" but it's really just mailpit. This can easily be removed and whatever breaks will need to be upgraded.

Reply:
The simple solution would be: `warden svc down --remove-orphans && warden svc up`.

Reply:
Yeah, I did like that and it worked. But I got confused that container is named mailhog, while we don't have such service and domain. So I would prefer removing this line